scripts/vagrantup.sh

```diff
@@ -0,0 +1,20 @@
+#!/bin/bash
+
+set -euo pipefail
+
+num_nodes=${CONTIV_NODES-2}
+num_masters=${CONTIV_MASTERS-1}
```

Comment:
Vagrantfile seems to be using CONTIV_NODES and CONTIV_MASTERS. Here you are leaving a few nodes